### PR TITLE
Redirs temp

### PR DESCRIPTION
**Pull request by Knulpinette**
I fucked up and forgot I was on a previous commit for the branch... haha
So you can check out everything in order (redirs, then redirs_temp) but merge first redirs_temp to redirs and then redirs to master.

### CODE REVIEW

File: srcs/utils/error_exit.c [hunk 1]
```diff
@@ -20,6 +24,11 @@ void	error_message(t_error code)
 		ft_putstr_fd("Wrong number of arguments.\n", STDERR_FILENO);
 	else if (code == NO_OLDPWD)
 		ft_putstr_fd("OLDPWD not set.\n", STDERR_FILENO);
+	/* can you check the input:     > test       to see if you have leaks or a segfault ? */
```
Comment (arieivs) [inline srcs/utils/error_exit.c:27]: why is this here, isn't this related to the seg fault when there's no command? Or is this something else?

Comment (Knulpinette) [reply srcs/utils/error_exit.c:27]: oh ! I actually corrected that. I forgot to get rid of the comment.
